### PR TITLE
feat: packet traces feature for Retina CLI

### PR DESCRIPTION
**Pull request by nddq**
# Description

Readding this WIP trace feature for Retina CLI to the repo. Also remove a bunch of Apache License since we are using MIT now.

## Related Issue

Will open an issue to keep track of development.

### CODE REVIEW

File: test/plugin/dns/main_linux.go [hunk 1]
```diff
@@ -1,17 +1,3 @@
-// Copyright 2022 The Inspektor Gadget authors
```
Comment (rbtr) [inline test/plugin/dns/main_linux.go:1]: this one probably needs to stay

File: test/utsummary/main.go [hunk 2]
```diff
@@ -1,17 +1,3 @@
-// Copyright 2019 The Go Cloud Development Kit Authors
```
Comment (rbtr) [inline test/utsummary/main.go:1]: this one too

**Review by rbtr — COMMENTED** [review]
A couple of these are (the original?) 3rd party licenses and should stay. The license of the file supersedes the default license of the repo

File: docs/CRDs/TracesConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# TraceConfiguration
+
+> **Note:** This feature is currently under experimental development.
```
Comment (vakalapa) [inline docs/CRDs/TracesConfiguration.md:3]: add a comment here that this is not implemented in bold headings ?

**Review by vakalapa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**